Projects/EditCoInsured/Sources/View/AlreadyExistingCoInsured/InsuredPeopleScreen.swift

```diff
@@ -9,6 +9,8 @@ struct InsuredPeopleScreen: View {
     @PresentableStore var store: EditCoInsuredStore
     @ObservedObject var vm: InsuredPeopleNewScreenModel
     @ObservedObject var intentVm: IntentViewModel
+    @EnvironmentObject private var editCoInsuredNavigation: EditCoInsuredNavigationViewModel
+    let onDisappear: () -> Void
```

Comment:
Recheck if we can get rid of this onDisappear. We should build logic that we don't need to do the chaining to upper level